Gulpfile.js

```diff
@@ -804,6 +805,77 @@ export function functional(done) {
   return done(result.status);
 }
 
+export async function getKafkaImageRepoTag() {
+  const manifest = loadPackageJson();
+  const extensionSettings = manifest.contributes.configuration;
+  const localSettings = extensionSettings.find((section) => section.title === "Local");
+  if (!localSettings)
+    throw new Error('Failed to find "Local" section in extension settings configuration');
+
+  const defaultKafkaImage =
+    localSettings.properties["confluent.localDocker.kafkaImageRepo"].default;
+  const defaultKafkaImageTag =
+    localSettings.properties["confluent.localDocker.kafkaImageTag"].default;
+  const defaultKafkaImageRepoTag = `${defaultKafkaImage}:${defaultKafkaImageTag}`;
+  // used by CI to capture the image to cache or load from the cache
+  console.log(defaultKafkaImageRepoTag);
+  return Promise.resolve(defaultKafkaImageRepoTag);
```

Comment:
Necessary to make this async and return a Promise since the gulp CLI [doesn't allow making synchronous tasks](https://gulpjs.com/docs/en/getting-started/async-completion#no-synchronous-tasks), and these tasks are used by CI instead of reimplementing package.json parsing logic.